### PR TITLE
[FEATURE]1652 Adding configurable verbosity for gfx-backend and rendy, removing log_gfx_device_level

### PR DESCRIPTION
**Pull request by Puciek**
## Description

This pr removes the, as I understand, now obsolete `log_gfx_device_level` from logger.rs, and instead provides two new optional config entries: `log_gfx_backend_level` and `log_gfx_rendy_level` which default to Warn.

## Additions

- Added optional `log_gfx_backend_level` and `log_gfx_rendy_level` to configuration

## Removals

- Removed obsolete `log_gfx_device_level` from configuration

## PR Checklist

By placing an x in the boxes I certify that I have:

- [X] Updated the content of the book if this PR would make the book outdated.
- [X] Added a changelog entry if this will impact users, or modified more than 5 lines of Rust that wasn't a doc comment.
- [X] Added unit tests for new code added in this PR.
- [X] Acknowledged that by making this pull request I release this code under an MIT/Apache 2.0 dual licensing scheme.

If this modified or created any rs files:

- [X] Ran `cargo +stable fmt --all`
- [X] Ran `cargo clippy --all --features "empty"`
- [X] Ran `cargo test --all --features "empty"`

### CODE REVIEW

File: src/logger.rs [hunk 1]
```diff
@@ -110,11 +113,21 @@ impl Logger {
             StdoutLog::Off => {}
         }
 
-        if let Some(log_gfx_device_level) = config.log_gfx_device_level {
-            logger.dispatch = logger
-                .dispatch
-                .level_for("gfx_device_gl", log_gfx_device_level);
-        }
+        let log_gfx_backend_level = config.log_gfx_backend_level.unwrap();
```
Comment (azriel91) [inline src/logger.rs]: Hiya, could you use `if let Some(log_gfx_backend_level) = config.log_gfx_backend_level { .. }`, and similar for `log_gfx_rendy_level`?
It's a valid scenario to construct `LoggerConfig` with them as `None`.

Thanks :bowing_man:!

Comment (Puciek) [reply src/logger.rs]: Sure thing, I'll do that!

**Comment by Puciek** [comment]
Will do today, was surprisingly busy end of year at work :)

**Comment by Puciek** [comment]
@azriel91 @distransient Thanks for the patience, updated the PR as per the comments.

**Pull request merged**